source/Outfit.cpp

```diff
@@ -118,17 +118,27 @@ const vector<string> Outfit::CATEGORIES = {
 
 void Outfit::Load(const DataNode &node)
 {
-	if(node.Size() >= 2)
+	if(node.Size() == 2)
 	{
 		name = node.Token(1);
 		pluralName = name + 's';
 	}
+	else if(node.Size() >= 3)
+	{
+		name = node.Token(2);
+		parentName = node.Token(1);
+		pluralName = name + 's';
+		isVariant = true;
+	}
```

Comment:
This isVariant boolean is also unnecessary. `if(!parentName.empty())` and `if(isVariant)` are functionally equivalent without the need of a new variable. And as I mentioned in my previous review, outfit variants should share the display name of their parent, so this should be something more like this:
```suggestion
	if(node.Size() >= 2)
	{
		name = node.Token(1);
		pluralName = name + 's';
		if(node.Size() >= 3)
			variantName = node.Token(2);
	}
```
This will require other changes elsewhere, such as the creation of a TrueName function that returns variantName if it exists, otherwise returning name. This function would then be used when a ship is saving its outfits so that we don't accidentally convert a variant outfit to its parent.